demo/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: nfs-provisioner
+  labels:
+    app: nfs-provisioner
+spec:
+  ports:
+    - name: nfs
+      port: 2049
+    - name: mountd
+      port: 20048
+    - name: rpcbind
+      port: 111
+    - name: rpcbind-udp
+      port: 111
+      protocol: UDP
+  selector:
+    app: nfs-provisioner
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: nfs-provisioner
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nfs-provisioner
+    spec:
+      containers:
+        - name: nfs-provisioner
+          image: quay.io/kubernetes_incubator/nfs-provisioner:v1.0.0
```

Comment:
Just for context all incubator projects so far have been using Quay.io or gcr.io. I don't care where things are hosted I offered to host this with the other Incubator projects on quay.io already. 

For example the node-feature-discovery projects likes Quay's image scanning features https://github.com/kubernetes-incubator/node-feature-discovery/issues/10

Docker hub is fine too. I really don't care but I think quay.io has compelling features.